packages/api/tests/endpoints/providers/get-provider.js

```diff
@@ -89,22 +115,22 @@ test('CUMULUS-912 GET with pathParameters and with an invalid access token retur
 test.todo('CUMULUS-912 GET with pathParameters and with an unauthorized user returns an unauthorized response');
 
 test('GET returns an existing provider', async (t) => {
-  const response = await request(app)
-    .get(`/providers/${t.context.testProvider.id}`)
+  const response = await request(t.context.app)
+    .get(`/providers/${t.context.testProvider.name}`)
     .set('Accept', 'application/json')
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
 
-  t.is(response.body.id, t.context.testProvider.id);
+  t.like(await translateApiProviderToPostgresProvider(response.body), t.context.testProvider);
```

Comment:
Is there a strong reason to use `like`?

Reply:
This is being used here largely because the timestamps are being auto-populated by the psql write. 

We're comparing (for example) :

{
  created_at: 2021-06-17T16:31:06.221Z,
  updated_at: 2021-06-17T16:31:06.221Z,
  name: 'provider08290',
  host: 'test-bucket',
  protocol: 's3',
}

with { name: 'provider08290', host: 'test-bucket', protocol: 's3' }